website/source/api/acl/policies.html.md

```diff
@@ -3,19 +3,17 @@ layout: api
 page_title: ACL Policies - HTTP API
 sidebar_current: api-acl-policies
 description: |-
-  The /acl/policy endpoints manage Consul's ACL Policies.
+  The /acl/policy endpoints manage Consul's ACL policies.
 ---
 
 -> **1.4.0+:**  The APIs are available in Consul versions 1.4.0 and later. The documentation for the legacy ACL API is [here](/api/acl/legacy.html)
 
 # ACL Policy HTTP API
 
 The `/acl/policy` endpoints [create](#create-a-policy), [read](#read-a-policy),
-[update](#update-a-policy), [list](#list-policies) and
-[delete](#delete-a-policy)  ACL policies in Consul.  
+[update](#update-a-policy), [list](#list-policies) and [delete](#delete-a-policy) ACL policies in Consul.
 
-For more information on how to setup ACLs, please see
-the [ACL Guide](https://learn.hashicorp.com/consul/advanced/day-1-operations/production-acls).
+For more information about ACLs, please see the [ACL Guide](/docs/guides/acl.html).
```

Comment:
Both links are going to the same place, is there a reason why you changed that @scalp42? I wonder if we prefer a direct link to learn or the redirect @hashicorp/consul-docs.

Reply:
Weird when I checked they were not going to the same place.

Reply:
If you don't want to wait for the feedback of the other team, just change it back and I will merge happily.

Reply:
should be good now @i0rek